proof/infoflow/RISCV64/Example_Valid_State.thy

```diff
@@ -1099,13 +1102,18 @@ lemma vs_lookup_s0_SomeD:
                      pool_for_asid_s0 asid_pools_of_s0 vspace_for_pool_def
               split: if_splits)+
 
+lemma pt_bits_left_max_minus_1_pageBitsForSize:
+  "pt_bits_left (max_pt_level - 1) = pageBitsForSize RISCVLargePage"
+  apply (clarsimp simp: pt_bits_left_def max_pt_level_def2)
+  done
```

Comment:
should this get moved somewhere higher?

Reply:
It's pretty specific, but I guess it would be useful if we have other situations where the different formulations meet. Any suggestions for a sensible location for it?

Reply:
I would leave it in the example -- the lemma people should be using in abstract proofs is already available higher up, and if we move the concrete one then people might feel the urge to complete the set and then we'd have hard coded the number of levels and relationships instead of keeping them generic.